cmd/observe/observe.go

```diff
@@ -177,14 +180,53 @@ more.`,
 
 	observerCmd.Flags().Var(filterVar(
 		"verdict", ofilter,
-		fmt.Sprintf("Show only flows with this verdict [%v, %v]", pb.Verdict_FORWARDED, pb.Verdict_DROPPED)))
+		fmt.Sprintf(
+			"Show only flows with this verdict [%v, %v, %v]",
+			pb.Verdict_FORWARDED, pb.Verdict_DROPPED, pb.Verdict_ERROR),
+	))
+	observerCmd.RegisterFlagCompletionFunc("verdict", func(_ *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
+		return []string{
+			pb.Verdict_FORWARDED.String(),
+			pb.Verdict_DROPPED.String(),
+			pb.Verdict_ERROR.String(),
```

Comment:
i think these two should use the same `[]string` slice, just one can do `strings.Join(",")`